owncloud/ios-app
# [fix/stuck-sync] Fix stuck actions after app termination / background activity

## Description
- fixes an issue where sync action events wouldn't be delivered when added to a core while it starts
- last selected bookmark no longer is auto-openend when the app is launched in the background by iOS
- improves durability of underlying SQLite databases

## How to test
- log into an account
- upload several images from the photo library
- force-quit the app
- wait a minute (optional / alternative check)
- launch app again and check that uploads finish and don't get stuck

## Types of changes
